packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, observable } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorsScale,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'quadrant'
+    })
+    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+
+    @attr({
+        attribute: 'orientation'
+    })
+    public orientation: WaferMapOrientation = WaferMapOrientation.top;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters = 4;
+
+    @attr({
+        attribute: 'show-die-labels'
+    })
+    public showDieLabels = true;
```

Comment:
## Background

Boolean attributes have a particular behavior, see the [mdn boolean attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes) topic

I encourage you to read and understand that topic. The summary is that the unique behavior of boolean attributes is that the absense of the attribute is the value false while the presence of any value means true.

i.e. `showDieLabels === false` in HTML
```html
<nimble-wafer></nimble-wafer>
```

i.e. `showDieLabels === true` in HTML
```html
<nimble-wafer show-die-labels></nimble-wafer>
```

## Requirements

Because of the above, all boolean values MUST default to a value of false. So if the desired default is that die labels are visible (which I agree sounds like the best default), then the logic for the attribute name must be inverted.

```suggestion
    @attr({
        attribute: 'die-labels-hidden',
        mode: 'boolean'
    })
    public dieLabelsHidden = false;
```

Notice the following changes made above:
1. The logic was inverted. We want the labels to be visible by default, ie with the html `<nimble-wafer>` and the user can opt-out of the default by adding the attribute `<nimble-wafer die-labels-hidden>`
2. For the boolean name we are using the convention `<name>-hidden` or `<name>-visible`. See the documented [attribute common name patterns](https://github.com/ni/nimble/blob/0fe8cc54f7bd444830e3974969519593b532b083/packages/nimble-components/CONTRIBUTING.md#attribute-common-name-patterns)
3. The attribute decorator was configured for [`mode: 'boolean'`](https://www.fast.design/docs/fast-element/defining-elements/#customizing-attributes)